src/components/scenes/LoginScene.js

```diff
@@ -71,15 +71,23 @@ class LoginSceneComponent extends React.PureComponent<Props, State> {
     getBackgroundImage(this.props.disklet, backgroundImageServerUrl, theme.backgroundImage)
       .then(backgroundImage => this.setState({ backgroundImage }))
       .catch(e => this.setState({ backgroundImage: theme.backgroundImage }))
-    const { YOLO_USERNAME, YOLO_PASSWORD } = ENV
-    if (YOLO_USERNAME != null && YOLO_PASSWORD != null && firstRun) {
+    const { YOLO_USERNAME, YOLO_PASSWORD, YOLO_PIN } = ENV
+    if (YOLO_USERNAME != null && (YOLO_PASSWORD != null || YOLO_PIN != null) && firstRun) {
       const { context, initializeAccount } = this.props
       firstRun = false
       setTimeout(() => {
```

Comment:
It look like we don't need this setTimeout anymore.
I've tested it and could not find any issues. 
So just remove the timeout and the rest of the changes are approved.